tests/ios/libraries/StaticLib/ObjcLib/BUILD.bazel

```diff
@@ -0,0 +1,10 @@
+load("@build_bazel_rules_ios//rules:framework.bzl", "apple_framework")
+
+apple_framework(
+    name = "ObjcLib",
+    platforms = {"ios": "9.0"},
+    vendored_static_libraries = [
```

Comment:
Did you consider to provide it a glob of headers instead - mainly, headers aren't tied to the notion of a static library. I think you could write more explict code - and we'd prevent injecting the `*/*.h` 

```
    vendored_static_libraries = [
    ]
    hdrs = ["Header1.h"],
    module_name="Some",
```

Reply:
I wasnt sure what the correct way to declare this type of target was. I saw we gather the headers in the `apple_library` as well, so i didn't add `hdrs` to the target. 

I think providing the headers for this pure static lib makes sense since there aren't multiple slices we need to select over. 

Like you said though if we remove the implicit collection of headers from the library macro then it might break assumptions users are making. Im happy to go that route though, let me know what you think is best.